iothub/service/src/Utilities/ClientApiVersionHelper.cs

```diff
@@ -5,16 +5,15 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// Holds the API version numbers required in data-plane calls to the service for <see cref="RegistryManager"/>,
-    /// <see cref="ServiceClient"/>, and <see cref="JobClient"/>.
+    /// <see cref="ServiceClient"/>, and <see cref="ScheduledJobsClient"/>.
```

Comment:
Same comment as before. Go ahead and delete these specific references to clients. "The API version used in all service requests."